.github/workflows/formatting-check.yml

```diff
@@ -2,6 +2,8 @@ name: Formatting Check
 on:
   pull_request:
     branches: [ master ]
+    paths-ignore:
+      - '.github/**'
```

Comment:
Interesting, how does this relate to when the Action is triggered?

Reply:
@gunnarmorling Ah well, I think I have to individually ignore the rest of the workflows because I think it is unnecessary to run the formatting check for any change in the rest of the workflow files. WDYT?

Reply:
OK, I'm confused :) How is all this related to the task described in DBZ-4058, which is to run the contributor check only once per PR?

Reply:
@gunnarmorling I think @ani-sha just spotted a deficiency in the logic (why running formatting check if `.github` dir is modified) and fixed it in this PR. The meat od the PR is the change `contributor-check.yml`.

Reply:
Yes, @jpechane, you got me right. @gunnarmorling this has nothing to do with the trigger of `contributor-check.yml`.

Reply:
Doh, of course. I completely missed that "Opened" change above, thinking this is the sole change. Makes sense now :)